package.json

```diff
@@ -58,7 +58,7 @@
     "@storybook/addon-links": "^7.4.0",
     "@storybook/addons": "^7.4.0",
     "@storybook/blocks": "^7.4.0",
-    "@storybook/react": "^7.4.0",
+    "@storybook/react": "^7.5.1",
```

Comment:
Never update just the on package when they have the same versioning, all the` @storybook` packages needs to be updated in tandem. Suggest doing this in a separate PR tho as to track eventual bugs or problems that might arise from it down the line.